### ISSUE TITLE
jQuery

### ISSUE THREAD

**Issue by brianburridge** [tgaeta/Assignments#60]
_This is assignment 59 and is due on Mar 26, 2015._

* Add a [jQuery plugin](http://plugins.jquery.com/) to your Pinterest app
* Be sure to direct me to where on your app you've added it, both in your code and what page it's running on at Heroku so I can see it work.

**Comment by tgaeta** [comment]
@brianburridge https://stark-brushlands-7818.herokuapp.com/
main page - change the size of the screen - made it responsive with two jquery plugins with fun animations (isotope, images_loaded plugin)

https://github.com/tgaeta/pinterest/tree/master/vendor/assets/javascripts

Located in Pins Index View:
```ERB
<% content_for :isotope do %>
    // initialize Isotope after all images have loaded
    var $mcontainer = $('#mcontainer').imagesLoaded( function() {
      $('#mcontainer').isotope({
        itemSelector: '.item',
        getSortData: {
          name: '.name',
        },
        // layout mode options
        masonry: {
          columnWidth: 50,
          gutter: 11.5
        }
      });
    });
<% end %>
```

**Comment by tgaeta** [comment]
ToDo: Fix navbar to be responsive; add media-queries, add modal for when a pin is clicked, the show popups.

**Comment by tgaeta** [comment]
@brianburridge fixed navbar and search, probably won't do media queries, but will try to do modal before we start projects for practice

**Comment by brianburridge** [comment]
:+1:
